.github/workflows/asv-benchmarking.yml

```
@@ -19,35 +22,53 @@ jobs:
         with:
           repository: UXARRAY/uxarray
           fetch-depth: 0
```

```suggestion
        with:
          repository: UXARRAY/uxarray
          ref: main
          fetch-depth: 0
```

Okay, the two `.git/` folders is weird, but wasn't actually the problem. This isn't working because it's a PR and geocat-comp uses forks instead of branching. To compensate, you can add this line here. This workflow isn't designed to work on pull request diffs, so nothing should change other than it shouldn't crash on PRs anymore.

---

See it running here: https://github.com/UXARRAY/uxarray/actions/runs/9118883464/job/25073008303

---

This is great, I would have never caught this.